dev-docs/bidders/aax.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: bidder
+title: Aax
+description: Prebid Media.net Bidder Adaptor
+biddercode: aax
+gdpr_supported: true  
+media_types: banner,native
```

Comment:
Could you add the `multiformat_supported` flag as well that was introduced in #3694
It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.